introspection.go

```diff
@@ -157,6 +157,7 @@ type RelayItemState struct {
 	ID                      uint32 `json:"id"`
 	RemapID                 uint32 `json:"remapID"`
 	DestinationConnectionID uint32 `json:"destinationConnectionID"`
+	Tomb                    bool   `json:"tomb"`
```

Comment:
Shouldn't the `TestServer` state tests be failing since the pre-state doesn't have any relay items, but the post-state has tombstoned relay items?